lms/views/api/assignments.py

```diff
@@ -36,6 +36,11 @@ def create(self):
         elif content_type == "file":
             url = f"canvas://file/course/{params['course_id']}/file_id/{params['content']['file']['id']}"
             extra = {"canvas_file": params["content"]["file"]}
+        elif content_type == "vitalsource":
+            # We still don't store details of VitalSource assignments.
+            # Don't send a real ID back to the frontend to avoid confusion.
+            return {"ext_lti_assignment_id": None}
```

Comment:
So what's the deal here? We're _not_ storing VitalSource assignments in the DB? But we'll have to start storing them before we can actually start using this new API?